orc/src/main/java/org/apache/iceberg/orc/OrcMetrics.java

```diff
@@ -114,6 +120,7 @@ private static Metrics buildOrcMetrics(final long numOfRows, final TypeDescripti
           columnSizes,
           valueCounts,
           nullCounts,
+          Maps.newHashMap(),
```

Comment:
Why not use null when metrics are missing?

Reply:
I think I missed updating this during refactoring. Will update!